### PR TITLE
finished storybook job card features

### LINKED ISSUE #21: Storybook: Card Job title Features Component
## User story

**Who:** **As a** user
**What:** **I want to** be able to see different job title features associated with the job title depending on the answers that I give in the quiz
**Why:** **so that I can** see detailed info about the job title.

## Acceptance criteria

Create a storybook for the **Card** Job title Features Component that meets what's illustrated below and highlighted in pink

Screenshot 2021-04-13 at 16 40 56
Screenshot 2021-04-13 at 16 42 53
Screenshot 2021-04-13 at 16 42 31
Screenshot 2021-04-13 at 16 42 06

## Implementation details

Implement knobs so that we can change the Job Feature Titles and content of the Job Features

Use the font [Roboto+Mono](https://fonts.google.com/specimen/Roboto+Mono?preview.text=What%20is%20GuideIT%3F&preview.text_type=custom) and the color ```#000000``` for the title, the bullet points, and the content.

Use this for inspiration https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_shapes_triangle-right

### PR DESCRIPTION
**Pull request by Enqira**
# Description

Component storybook for right result job card features, with 3 bullet points. This component is called with one prop, the branch name,: Full stack development, UX design or data analysis. Depending on the result and the clicked bullet point, the user will see the needed info that is stored in an array.

Fixes #21 

# How to test?

command: npm run storybook, and click in the bullet points, you can change the control but typing a brach name and see different results
# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/ResultJobFeatures/ResultJobFeatures.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            onClick={() => handleSecondClick(secondRef)}
+            onKeyDown={() => handleSecondClick(secondRef)}
+          >{`A day in the life of a ${jobTitle}`}</div>
+        </div>
+        <div
+          className="job-features-description"
+          ref={secondRef}
+          style={{ display: secondStyle }}
+        >
+          <p>{`Interested in finding our what a ${jobTitle} does and how it looks like to be one?`}</p>
```
Comment (TITHI1244) [inline src/client/components/ResultJobFeatures/ResultJobFeatures.js:191]: A small typo "...finding out..." :)

**Review by TITHI1244 — COMMENTED** [review]
Super nice job @Enqira with precise comments and easy-to-read code. Thank you from team 4 :)

File: src/client/components/ResultJobFeatures/ResultJobFeatures.js (hunk 1 above)
Comment (Enqira) [reply src/client/components/ResultJobFeatures/ResultJobFeatures.js:191]: Good eje!!

**Comment by Enqira** [comment]
Thank you @TITHI1244 , I did implemented the changes

**Review by cph-kiwi — APPROVED** [review]
Wow, really impressive. Nice work.

**Review by TITHI1244 — APPROVED** [review]

**Pull request merged**